cosmos-gui/conf/cosmos-gui.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Although using a .json file is perfectly fine, a option to be able to include comments is using a js file exporting the object as you would in any other module. Take a look for example at: https://github.com/telefonicaid/IoT-STH/blob/master/config.js I leave it 100% at your discretion... ;)